kwyk/cli.py

```diff
@@ -54,21 +55,26 @@ def predict(*, infiles, outprefix, model, n_samples, batch_size, save_variance,
     savedmodel_path = _models[model]
     predictor = _get_predictor(savedmodel_path)
     for infile in infiles:
-        _predict(infile, outprefix, predictor, n_samples, batch_size, save_variance, save_entropy, overwrite, atlocation)
+        _predict(infile, outprefix, predictor, n_samples, batch_size, save_variance, save_entropy, overwrite, atlocation, base)
 
 
-def _predict(infile, outprefix, predictor, n_samples, batch_size, save_variance, save_entropy, overwrite, atlocation):
+def _predict(infile, outprefix, predictor, n_samples, batch_size, save_variance, save_entropy, overwrite, atlocation, base):
     _orig_infile = infile
 
     # Are there other neuroimaging file extensions with multiple periods?
     if infile.lower().endswith('.nii.gz'):
         outfile_ext = '.nii.gz'
     else:
         outfile_ext = Path(infile).suffix
-    outfile_stem = outprefix
+    outfile_stem = outprefix + Path(infile).name.replace(outfile_ext, "")
```

Comment:
one can use `Path.stem` here.

```suggestion
    outfile_stem = outprefix + Path(infile).stem
```

Reply:
will it support both 'nii.gz' and 'nii' ?

Reply:
oops, it will not. nice catch.